q2_demux/_format.py

```diff
@@ -18,6 +18,17 @@ class EMPMultiplexedDirFmt(model.DirectoryFormat):
         r'barcodes.fastq.gz', format=FastqGzFormat)
 
 
+class PairedEMPMultiplexedDirFmt(model.DirectoryFormat):
```

Comment:
Oh god, these names are so bad. I see that they're based on what was in there for the single-end read stuff - do we want to deal with these names now, or some other time?